common/src/main/kotlin/streams/utils/JSONUtils.kt

```diff
@@ -33,6 +54,42 @@ fun Point.toStreamsPoint(): StreamsPoint {
     }
 }
 
+fun Map<String, Any>.toMapValue(): MapValue {
```

Comment:
should we maybe use `org.neo4j.values.virtual.MapValueBuilder` instead?

Reply:
done